src/doc/gui_manual/Operators/OperatorTypes/ICS/LimitCycle_operator.rst

```diff
@@ -0,0 +1,110 @@
+.. _Limit Cycle operator:
+
+Limit Cycle operator
+~~~~~~~~~~~~~~~~~~~~
+
+The Limit Cycle Operator detects limit cycles within a vector field. Integral
+curves are seeded at a Poincaré section and integrated through the vector
+field. Curves that return to the Poincaré section indicate a limit cycle, and
+the integration of the curve will stop. Some integral curves will not return
+to the Poincaré section, which can be terminated according to termination
```

Comment:
Suggest slight re-wording...

Those integral curves that do not return to the Poincare section are terminated according to separate termination criteria.